### DISCUSSION TITLE
Vaultwarden LXC - alpine uppgrade web vault missing

### DISCUSSION THREAD

**Discussion by damadorPL** [tteck/Proxmox#1947] [Help Needed] [answered]
Previously used Vaultwarden LXC 

"To Update Vaultwarden, Web-Vault or Set the Admin Token, run the command below (or type update) in the LXC Console."

in alpine version 

To Update Alpine-Vaultwarden, or Set the Admin Token, run the command below in the LXC Console.

There is no option to update the web vault 

any hints ?

**Comment by tteck** [comment, marked as answer]
Both are updated via `apk update && apk upgrade`
